services/auth_service.py

```
@@ -47,24 +48,36 @@ def signup_user(data: UserSignUp):
 def login_user(data: UserLogin):
-    users = get_users()
-    user = next((u for u in users if u["email"] == data.username), None)
+    con = get_db_connection()
+    try:
+        with con.cursor() as cursor:
+            check_sql = "SELECT user_id,email, password, nickname, profile_image_url FROM users where email = %s"
+            cursor.execute(check_sql, (data.username,))
+            user_record = cursor.fetchone()
+            print(user_record)
+            if user_record is None:
+                raise HTTPException(status_code=401, detail="유저 정보(이메일 혹은 비밀번호)가 없습니다.")
```

Ruff (0.14.13)

Abstract `raise` to an inner function

(TRY301)